app/controllers/health_checks_controller.rb

```diff
@@ -16,7 +16,7 @@ def show
   def new
     @health_check = HealthCheck.new
     authorize @health_check
-    @clinics = Clinic.all
+    @clinics = Clinic.order("name ASC")
```

Comment:
Could you include the ".all" next time? .order should be performed on a collection, not a class.